# Задание 1

## Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде .Результаты: 1)Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 3) Конверсии : conv1 = 1.05% conv2 = 1.47% .

In [42]:
import numpy as np
import plotly.graph_objects as go

H0: Между конверсиями в двух группах нет статистически значимых различий

H1: Между конверсиями в двух гркппах есть различия.

In [57]:
a = 0.0147 - 0.0105 - (1.96 * np.sqrt(0.0105 * ((1- 0.0105) / 15550) + 0.0147 * ((1 - 0.0147) / 15550)))
a

0.0017210902166048432

In [58]:
a * 100 

0.1721090216604843

In [59]:
b = 0.0147 - 0.0105 + (1.96 * np.sqrt(0.0105 * ((1- 0.0105) / 15550) + 0.0147 * ((1 - 0.0147) / 15550)))
b

0.0066789097833951545

In [60]:
b * 100

0.6678909783395155

#### Вывод: Интервал [0.17% ; 0.66%] Т.к ноль не попадает в интервал есть статистически значимой разнца между группами.

# Задание 2


## Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

Конверсия колличественный тип данных -> распределение нормальное -> зависимые -> если две группы, то T-критерий Стьюдента; если 3 и более, то дисперсионный анализ.